deno-runtime/lib/accessors/mod.ts

```diff
@@ -189,7 +193,23 @@ export class AppAccessors {
                 getUserReader: () => this.proxify('getReader:getUserReader'),
                 getNotifier: () => this.proxify('getReader:getNotifier'),
                 getLivechatReader: () => this.proxify('getReader:getLivechatReader'),
-                getUploadReader: () => this.proxify('getReader:getUploadReader'),
+                getUploadReader: () => ({
+                    _proxy: this.proxify('getReader:getUploadReader'),
+                    getById(id: string) {
+                        return this._proxy.getById(id);
+                    },
+                    // Convert the Uint8Array to a Buffer 
+                    getBufferById(id: string) {
+                        const result = this._proxy.getBufferById(id);
+                        return Buffer.from(result);
+                    },
+                    // Convert the Uint8Array to a Buffer 
+                    getBuffer(upload: IUpload) {
+                        const result = this._proxy.getBuffer(upload);
+                        return Buffer.from(result);
```

Comment:
```suggestion
                    async getBufferById(id: string) {
                        const result = await this._proxy.getBufferById(id);
                        return Buffer.from(result);
                    },
                    // Convert the Uint8Array to a Buffer 
                    async getBuffer(upload: IUpload) {
                        const result = await this._proxy.getBuffer(upload);
                        return Buffer.from(result);
```

Calls to proxy methods always return a promise